common.js

```diff
@@ -1686,6 +1686,445 @@ const allSoloCars = {
+    },
+    'Sentra (B13 chassis; 2.0 L)': {
+      '1991': ['ep'],
+      '1992': ['ep'],
+      '1993': ['ep'],
+      '1994': ['ep']
+    },
+    'Sentra & 200SX (B14 chassis)': {
+      '1995': ['ep'],
+      '1996': ['ep'],
+      '1997': ['ep'],
+      '1998': ['ep'],
+      '1999': ['ep']
+    },
+    'Sentra (B15 chassis)': {
+      '2002': ['dsp'],
+      '2003': ['dsp'],
+      '2004': ['dsp'],
+      '2005': ['dsp'],
+      '2006': ['dsp']
+    },
+    'Sentra (B16 chassis)': {
+      '2007': ['dsp'],
+      '2008': ['dsp'],
+      '2009': ['dsp'],
+      '2010': ['dsp'],
+      '2011': ['dsp'],
+      '2012': ['dsp']
+    },
+    'Versa': {
+      '2007': ['hs', 'fsp'],
+      '2008': ['hs', 'fsp'],
+      '2009': ['hs', 'fsp'],
+      '2010': ['hs', 'fsp', 'ep'],
+      '2011': ['hs', 'fsp', 'ep'],
+      '2012': ['hs', 'fsp', 'ep'],
+      '2013': ['hs', 'fsp', 'ep'],
+      '2014': ['hs', 'fsp', 'ep'],
+      '2015': ['hs', 'fsp', 'ep'],
+      '2016': ['hs', 'fsp'],
+      '2017': ['hs'],
+      '2018': ['hs'],
+      '2019': ['hs'],
+      '2020': ['hs'],
+    },
+    '1200': {
+      'all': ['hs', 'fsp']
+    },
+    '210': {
+      'all': ['fsp']
+    },
+    '210 (1397 & 1488 cc)': {
+      '1979': ['dp'],
+      '1980': ['dp'],
+      '1981': ['dp'],
+      '1982': ['dp']
+    },
+    '210 (B310 chassis; 1.4 L)': {
+      '1978': ['dp'],
+      '1979': ['dp'],
+      '1980': ['dp'],
+      '1981': ['dp'],
+      '1982': ['dp']
+    },
+    'B210': {
+      'all': ['fsp']
+    },
+    '1500 & 1600 Roadster': {
+      'all': ['hs']
+    },
+    '1500 (SPL 310), 1600 (SPL 311/311U), & 2000 (SRL 311) Roadster': {
+      'all': ['dp']
+    },
+    '210 & B-210': {
+      'all': ['hs']
+    },
+    '310': {
+      'all': ['hs', 'fsp']
+    },
+    '510': {
+      '1968': ['fsp', 'hs'],
+      '1969': ['fsp', 'hs'],
+      '1970': ['fsp', 'hs'],
+      '1971': ['fsp', 'hs'],
+      '1972': ['fsp', 'hs'],
+      '1973': ['fsp', 'hs'],
+      '1978': ['fsp', 'hs'],
+      '1979': ['fsp', 'hs'],
+      '1980': ['fsp', 'hs'],
+      '1981': ['fsp', 'hs']
+    },
+    '510 (A10 chassis)': {
+      '1979': ['dp'],
+      '1980': ['dp'],
+      '1981': ['dp']
+    },
+    '510 (PL510 chassis; 1.6 L, 1.8 L, & 2.0 L)': {
+      '1968': ['dp'],
+      '1969': ['dp'],
+      '1970': ['dp'],
+      '1971': ['dp'],
+      '1972': ['dp'],
+      '1973': ['dp']
+    },
+    '510 (PL510) (1595 cc)': {
+      'all': ['dp']
+    },
+    '610': {
+      'all': ['fsp', 'hs'],
+      '1973': ['dp'],
+      '1974': ['dp'],
+      '1975': ['dp'],
+      '1976': ['dp']
+    },
+    '710': {
+      'all': ['fsp', 'hs'],
+      '1974': ['dp'],
+      '1975': ['dp'],
+      '1976': ['dp'],
+      '1977': ['dp']
+    },
+    '720 (2WD)': {
+      '1980': ['dp'],
+      '1981': ['dp'],
+      '1982': ['dp'],
+      '1983': ['dp'],
+      '1984': ['dp'],
+      '1985': ['dp'],
+      '1986': ['dp']
+    },
+    '810': {
+      'all': ['hs'],
```

Comment:
Remove `all` and add `hs` to all model years.

Reply:
Done!